src/ucp/core/ucp_context.c

```diff
@@ -329,6 +329,12 @@ static ucs_config_field_t ucp_config_table[] = {
    "endpoint.",
    ucs_offsetof(ucp_config_t, ctx.proto_indirect_id), UCS_CONFIG_TYPE_ON_OFF_AUTO},
 
+  {"ERROR_HANDLER_DELAY", "0us",
+   "Time period between error has been detected and handled (0 - disabled).\n"
```

Comment:
Artificial delay between detecting an error and processing it (0 - disabled). Used for testing purposes.